spec/fixtures/files/protocolio/normalized_single_protocol.json

```diff
@@ -0,0 +1,406 @@
+{
```

Comment:
This file and the one below should probably reside in `protocols_io/v3/normalized_single_protocol.json`. Mind the missing `s` and the API number.